dev/ci/render-pr-preview.sh

```diff
@@ -0,0 +1,176 @@
+# Should be in formats:
+# - https://github.com/sourcegraph/sourcegraph
+# - git://github.com/sourcegraph/sourcegraph.git
+repo_url="${BUILDKITE_PULL_REQUEST_REPO}"
+
+if [ -z "${repo_url}" ]; then
+  repo_url=$(git config --get remote.origin.url)
+fi
+
+while getopts 'b:r:d' flag; do
+  case "${flag}" in
+    d) is_deleting="true" ;;
+    b) branch_name="${OPTARG}" ;;
+    r) repo_url="${OPTARG}" ;;
+    *)
+      print_usage
+      exit 1
+      ;;
+  esac
+done
+
+if [[ "$repo_url" =~ ^(https|git):\/\/github\.com\/(.*)$ ]]; then
+  owner_and_repo="${BASH_REMATCH[2]//\.git/}"
+else
+  echo "Couldn't find owner_and_repo"
+  exit 1
+fi
+
+render_api_key="${RENDER_COM_API_KEY}"
+render_owner_id="${RENDER_COM_OWNER_ID}"
+pr_number="${BUILDKITE_PULL_REQUEST}"
+github_api_key="${GITHUB_TOKEN}"
+
+if [[ -z "${render_api_key}" || -z "${render_owner_id}" ]]; then
+  echo "RENDER_COM_API_KEY or RENDER_COM_OWNER_ID is not set"
+  exit 1
+fi
+
+pr_preview_app_name="sg-web-${branch_name}"
+
+renderServiceId=$(curl -sS --request GET \
+  --url "https://api.render.com/v1/services?limit=1&type=web_service&name=$(urlencode "$pr_preview_app_name")" \
+  --header 'Accept: application/json' \
+  --header "Authorization: Bearer ${render_api_key}" | jq -r '.[].service.id')
+
+if [ "${is_deleting}" = "true" ]; then
+  if [ -z "${renderServiceId}" ]; then
+    echo "Render app not found"
+
+    exit 0
+  fi
+
+  curl -sSf -o /dev/null --request DELETE \
+    --url "https://api.render.com/v1/services/${renderServiceId}" \
+    --header 'Accept: application/json' \
+    --header "Authorization: Bearer ${render_api_key}"
+
+  echo "Render app is deleted!"
+
+  exit 0
+fi
+
+if [ -z "${renderServiceId}" ]; then
+  pr_preview_url=$(curl -sSf --request POST \
+    --url https://api.render.com/v1/services \
+    --header 'Accept: application/json' \
+    --header 'Content-Type: application/json' \
+    --header "Authorization: Bearer ${render_api_key}" \
+    --data "
+    {
+        \"autoDeploy\": \"yes\",
+        \"envVars\": [
+            {
+                \"key\": \"ENTERPRISE\",
+                \"value\": \"1\"
+            },
+            {
+                \"key\": \"NODE_ENV\",
+                \"value\": \"production\"
+            },
+            {
+                \"key\": \"PORT\",
+                \"value\": \"3080\"
+            },
+            {
+                \"key\": \"SOURCEGRAPH_API_URL\",
+                \"value\": \"https://k8s.sgdev.org\"
+            },
+            {
+                \"key\": \"SOURCEGRAPHDOTCOM_MODE\",
+                \"value\": \"false\"
+            },
+            {
+                \"key\": \"WEBPACK_SERVE_INDEX\",
+                \"value\": \"true\"
+            }
+        ],
+        \"serviceDetails\": {
+            \"pullRequestPreviewsEnabled\": \"no\",
+            \"envSpecificDetails\": {
+                \"buildCommand\": \"yarn install && dev/ci/yarn-build.sh client/web\",
```

Comment:
`yarn install` is included into `yarn-build.sh`

```suggestion
                \"buildCommand\": \"dev/ci/yarn-build.sh client/web\",
```